test/test-server/ccloud_handlers.go

```diff
@@ -51,6 +51,7 @@ var (
 const (
 	serviceAccountID           = int32(12345)
 	serviceAccountResourceID   = "sa-12345"
+	groupMappingResourceId     = "pool-abc"
```

Comment:
Either change all the other variables here to use "Id", or call this variable "ID". Consistency is always the better approach.